packages/sources/bitgo-reserves/src/transport/reserves.ts

```
@@ -89,7 +98,17 @@ export const httpTransport = new ReservesHttpTransport({
 
     const verifier = crypto.createVerify('sha256')
     verifier.update(payload.data)
-    if (!verifier.verify(httpTransport.pubkey, payload.dataSignature, 'base64')) {
+    let verified: boolean
+    try {
+      verified = verifier.verify(
+        params[0].client == 'gousd' ? httpTransport.pubkey : getCreds(params[0].client)[1],
+        payload.dataSignature,
+        'base64',
+      )
+    } catch (err) {
+      verified = false
+    }
```

Do we really want to silently drop the error about the missing environment variable?

---

We are not, it will be logged in prepareRequests

---

Oh, so it's actually not possible for it to throw here because then it would have already thrown earlier and we would have never reached here?
If I understood that correctly, would be good to at least add a comment here.
But do we even need the try/catch here then?

---

Yes, removed